bluemira/equilibria/analysis.py

```diff
@@ -698,7 +698,7 @@ def plot_compare_separatrix(
         show=True,  # noqa: FBT002
     ):
         """
-        Plot separatrices of input equilibria and reference equilibria.
+        Plot seperatricies of input equilibria and reference equilibria.
```

Comment:
Should this be, dunno though:
```suggestion
        Plot separatricies of input equilibria and reference equilibria.
```

Reply:
L197 of diagnostics too

Reply:
Nice spot, I was wondering why my spell checker was not working on the plural.